.circleci/config.yml

```diff
@@ -19,13 +19,14 @@ jobs:
     working_directory: ~/repo
     docker:
       - image: circleci/node:10.0
-      - image: gtaschuk/testrpc
+      - image: trufflesuite/ganache-cli:latest
+        command: --gasLimit 18000000
```

Comment:
@gtaschuk This isn't changing the ganache-cli gas limit, which is why the tests are failing. Can you see what I've done wrong?